### PR TITLE
Feature/pubsub integration fix conflict

### PR DESCRIPTION
**Pull request by sandisunandar99**
## Overview 

- fixed conflict code with before
- fixing audit npm 
- and code changing same with before

### CODE REVIEW

File: models/index.js [hunk 1]
```diff
@@ -47,7 +47,6 @@ const register = (server, options, next) => {
       Specimen: db.model('Specimen'),
       UserDevice: db.model('UserDevice'),
       LogSelfReport: db.model('LogSelfReport'),
-      LogQueue: db.model('LogQueue'),
```
Comment (firmanJS) [inline models/index.js:50]: iyeu naha di hapus di master ges ka up soalna

File: services/index.js [hunk 2]
```diff
@@ -29,7 +29,6 @@ const register = (server, options, next) => {
     require('./inspection_support'),
     require('./search'),
     require('./cases_other'),
-    require('./queue'),
```
Comment (firmanJS) [inline services/index.js:32]: iyeu oge mang naha di hapus ??

**Review by firmanJS — APPROVED** [review]

File: services/index.js (hunk 2 above)
Comment (sandisunandar99) [reply services/index.js:32]: oh wkwkwk, sory sugan teh can perlu,, tadi pas solved conflic ka hapus

**Comment by firmanJS** [comment]
gass

**Pull request merged**